Shared/SortableTableLibrary/sortableTable.js

```diff
@@ -105,44 +104,44 @@ function clickedInternal(event, clickdobj) {
 
 	if (sortableTable.currentTable.showEditCell != null) {
 		var cellelement = event.target.closest("td");
-    var rowelement = event.target.closest("tr");
-    let regex=new RegExp("^r([0-9]+)"+DELIMITER+"([a-zA-Z0-9]+)"+DELIMITER+"(.*)")
-    let match=cellelement.id.match(regex);
-    var rowno = match[1];
-    var columnno = null; // Not used anymore
-    var tableid = match[2];
-    var columnname=match[3]
-    var str = "";
-    var rowdata = sortableTable.currentTable.getRow(rowno);
-    var coldata = rowdata[columnname];
-
-    sortableTable.edit_rowno = rowno;
-    sortableTable.edit_row = rowdata;
-    sortableTable.edit_columnno = columnno;
-    sortableTable.edit_columnname = columnname;
-    sortableTable.edit_tableid = tableid;
-    sortableTable.edit_celldata = coldata;
-    var estr=sortableTable.currentTable.showEditCell(coldata,rowno,rowelement,cellelement,columnname,columnno,rowdata,coldata,tableid);
-    if(estr!==false){
-      str += "<div id='input-container' style='flex-grow:1'>";
-      str += estr;
-      str += "</div>";
-      str += "<img id='popovertick' class='icon' src='../Shared/icons/Icon_Tick.svg' onclick='updateCellInternal();'>";
-      str += "<img id='popovercross' class='icon' src='../Shared/icons/Icon_Cross.svg' onclick='clearUpdateCellInternal();'>";
-      var lmnt = cellelement.getBoundingClientRect();
-      var popoverelement = document.getElementById("editpopover");
-
-      popoverelement.innerHTML = str;
-      var popoveredit = document.getElementById("popoveredit");
-      var xscroll = window.pageXOffset;
-      var yscroll = window.pageYOffset;
-
-      popoverelement.style.left = Math.round(lmnt.left+xscroll)+"px";
-      popoverelement.style.top = Math.round(lmnt.top+yscroll)+"px";
-      popoverelement.style.minHeight = (Math.round(lmnt.height)-5)+"px";
-      popoverelement.style.maxWidth = "fit-content";
-      popoverelement.style.display = "flex";
-    }
+		var rowelement = event.target.closest("tr");
```

Comment:
Its hard to see whats really changed when the indentation are changed across the file

Reply:
is it ok to merge these fixes and re-do some testing?
@a17andby @a17jacsv